WcaOnRails/spec/features/set_locale_spec.rb

```diff
@@ -0,0 +1,24 @@
+# frozen_string_literal: true
+require "rails_helper"
+
+RSpec.feature "Set the locale" do
+  context "As a visitor" do
+    let(:user) { FactoryGirl.create :user }
+
+    scenario "visiting the home page and changing the locale" do
+      visit "/"
+      expect(I18n.locale).to eq I18n.default_locale
+      visit update_locale_path("fr")
+      visit "/"
+      expect(I18n.locale).to eq :fr
+    end
+
+    scenario "signing in updates to the preferred_locale" do
+      user.preferred_locale = "fr"
+      user.save!
+      sign_in user
+      visit "/"
+      expect(I18n.locale).to eq user.preferred_locale&.to_sym
```

Comment:
You could just go for `:fr` here, whatever you prefer. I'm just kind of surprised by this `&.` when we know what exactly should be here.